features/step_definitions/basic_steps.rb

```diff
@@ -27,6 +27,6 @@
   click_on element
 end
 
-Then("I am on Sign up page") do
-  expect(current_path).to eq new_user_registration_path
-end
\ No newline at end of file
+When("I select {string} from {string}") do |option, selection|
+  select option, from: `product.#{selection}`
```

Comment:
is it really working with the backticks?

Reply:
yes it is :)